volume/struct.go

```diff
@@ -50,6 +50,13 @@ const (
 	DistDispStripe
 )
 
+// Volume list
+// Represent the list of volume during GetVol request
+type Volume_list struct {
```

Comment:
A simple map[string]string should suffice for this. No need to create a whole new type.